neofetch

```diff
@@ -2387,6 +2393,10 @@ get_gpu() {
                         [[ -z "$(trim "$gpu")" ]] && gpu="Intel Integrated Graphics"
                     ;;
 
+                    *"MCST"*)
+                        gpu="${gpu/*MCST*MGA2*/MCST MGA2}"
```

Comment:
Otherwise it will return garbage with pci device number in old versions of distro.